mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -335,6 +335,26 @@ public void testInsertFromSelectWithOrderBy() throws IOException {
     HiveIcebergTestUtils.validateData(table, records, 0);
   }
 
+  @Test
+  public void testInsertFromSelectWithProjection() throws IOException {
+    Assume.assumeTrue("Tez write is not implemented yet", executionEngine.equals("mr"));
+
+    Table table = testTables.createTable(shell, "customers", HiveIcebergStorageHandlerTestUtils.CUSTOMER_SCHEMA,
+        fileFormat, ImmutableList.of());
+    testTables.createTable(shell, "orders", ORDER_SCHEMA, fileFormat, ORDER_RECORDS);
+
+    shell.setHiveSessionValue("hive.vectorized.execution.enabled", false);
```

Comment:
Vectorization was already disabled in TestHiveShell.initializeConf? Do we need to disable it again?

Reply:
Yeah, I have missed this.
Thanks for catching!